### DISCUSSION TITLE
Allow url-encoded characters (e.g. German umlauts) in slug for dynamic pages?

### DISCUSSION THREAD

**Discussion by Alkalurops** [flareact/flareact#138] [Ideas]
At the moment this an access to this URL https://de.wikipedia.org/wiki/München would result in this error message:
`could not find wiki/M%C3%BCnchen/index.html in your content namespace`.

But I'd like it to work. 😃

**Comment by traffisco** [comment]
Use latest flareact release

**Reply by Alkalurops** [reply]
I updated to `flareact@1.1.0` but it still says: `could not find M%C3%BCnchen/index.html in your content namespace` 😿

**Reply by traffisco** [reply]
Try canary.
Was fixed in #105

**Reply by Alkalurops** [reply]
It works! Excellent! Thank you 😃
